AxonDeepSeg/visualization/get_masks.py

```diff
@@ -18,9 +18,13 @@ def get_masks(path_prediction):
     
     # get main path
     path_folder, file_name = os.path.split(path_prediction)
+
     tmp_path = path_prediction.split('_seg-axonmyelin')
 
     # save the masks
+    if tmp_path[0].endswith('.png'):
```

Comment:
Makeing the change above, this can be changed to:

```
if '.png' in tmp_path:
    (...)
```
(there is also a function called `os.path.splitext()` that can be used to split the root and extension of the path.

Reply:
Is there a reason why you feel that the `endswith` string method should be avoided?

I feel the way I implemented is "safer", as within the if statement I then remove the last four string characters. Hypothetically, using your `if` statement, if the string contains a ".png" sequence within the string (not at the end), it will still pass, and behave strangely.

Reply:
Just a suggestion, feel free to keep as it is, however the way that the extension is being captured isn't the recommended one (a better approach would be to use the correct splits from the `os.path`) instead of doing manual string manipulation.

Reply:
I agree, that makes sense, thanks @perone !